liquibase-core/src/main/java/liquibase/sqlgenerator/core/CreateTableGenerator.java

```diff
@@ -58,7 +59,13 @@ public Sql[] generateSql(CreateTableStatement statement, Database database, SqlG
             buffer.append(statement.getTableType().toUpperCase());
         }
 
-        buffer.append("TABLE ").append(database.escapeTableName(statement.getCatalogName(),
+        buffer.append("TABLE ");
+
+        if (database.supportsCreateIfNotExists(StoredProcedure.class)) {
```

Comment:
Hey, StoredProcedure was an example that Nathan used as another ways that we can use this method. So here you should be passing Table.class as a parameter (and then change all the other classes that are testing against it). So if in the future we change the stored procedure logic to add "if exists", then we will add it ;)

Reply:
Thanks!